src/main/java/pl/touk/flink/ignite/dialect/IgniteDialect.java

```diff
@@ -31,9 +30,72 @@ public String getLimitClause(long l) {
         return "LIMIT " + l;
     }
 
+    @Override
+    public String quoteIdentifier(String identifier) {
+        return "\"" + identifier + "\"";
+    }
+
     @Override
     public Optional<String> defaultDriverName() {
         return Optional.of("org.apache.ignite.IgniteJdbcThinDriver");
     }
 
+    // WARNING code below was taken from PostgresDialect, it can turn out to be wrong
+
+    private static final int MAX_TIMESTAMP_PRECISION = 6;
```

Comment:
Move constants at the beginning of the class.

Reply:
And precision seems to be 9: https://ignite.apache.org/docs/latest/sql-reference/data-types#timestamp. Write a test to make sure it's true?

Reply:
done, without test